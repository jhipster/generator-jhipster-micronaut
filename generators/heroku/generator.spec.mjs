import expect from 'expect';

import { helpers, lookups } from '#test-utils';

describe('SubGenerator heroku of micronaut JHipster blueprint', () => {
  describe('run', () => {
    let result;
    before(async function () {
      result = await helpers
        .create('jhipster:heroku')
        .withOptions({
          reproducible: true,
          defaults: true,
          blueprint: 'micronaut',
          baseName: 'jhipster',
        })
        .withLookups(lookups)
        .run();
    });

    it('should succeed', () => {
      expect(result.getStateSnapshot()).toMatchSnapshot();
    });
  });
});
