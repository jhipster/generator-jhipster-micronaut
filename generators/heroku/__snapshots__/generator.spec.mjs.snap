// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SubGenerator heroku of micronaut JHipster blueprint run should succeed 1`] = `
Object {
  ".yo-rc.json": Object {
    "stateCleared": "modified",
  },
}
`;
